styles/discography.css

```diff
@@ -0,0 +1,54 @@
+body {
+    background-color: #fffde0;
+    font-family: georgia;
+    color: #433c5e;
+}
+
+footer {
+    background-color: #ebc1d2;
+    border: solid;
+    border-color: #b98eb1;
+}
+
+h1 {
+    text-align: center;
+}
+
+ol img {
+    width: 300px;
+    height: 300px;
+    
+    width: 400px;
+    height: 400px;
+    background-color: #fefa99;
+    border: 2px solid #433c5e;;
+    padding: 7px;
+}
+.id {
+    float: left;
+}
+#Jolene {
```

Comment:
Totally valid to use capitalized class and id names, but the convention is to use lower case letters and kebab case for separators, e.g. `.example-cls`